src/tests/krate.rs

```diff
@@ -414,6 +414,69 @@ fn exact_match_on_queries_with_sort() {
     assert_eq!(json.crates[3].name, "other_sort");
 }
 
+#[test]
+fn loose_search_order() {
```

Comment:
This test is extremely long given how little code is being changed. Do you think this might be retesting things that are already tested elsewhere? Would it be worth trying to narrow the scope a bit to make it more clear what's being tested?

Reply:
General sort order is being addressed in other tests. I could reduce the number of creates to simplify the test. The basic idea is that we are now including crates that are partial matches. One of the previous comets about this PR was that the change should't impact current ordering so I tried to capture a few pieces of information that go into the current ordering like `_` working like white space or description matching superseding this new search. I can remove the non-matching crate and the extra partial match to shorten things up?